docs/remote-caching.md

```diff
@@ -229,7 +229,7 @@ Note however that there is small performance penalty for regularly checking the
 
 ## Alternatives
 
-An alternative to using remote caching is to use [remote runners](./remote-runners.md) (a commercial version of remote runners is [Earthly Satellites](cloud/satelllites.md)). Remote runners execute the build remotely, and this allows the cache to be located in close proximity to the execution, which is very efficient. Satellites are also significantly easier to set up, as the caching just works and there is no need for additional experimentation.
+An alternative to using remote caching is to use [remote runners](./remote-runners.md). Remote runners execute the build remotely, and this allows the cache to be located in close proximity to the execution, which is very efficient. Satellites are also significantly easier to set up, as the caching just works and there is no need for additional experimentation.
```

Comment:
We jump from talking about remote runners to talking about Satellites. Are we meaning to talk about both? If so, we should clarify that Satellites are remote runners hosted by Earthly.

Reply:
Yeah, I should fix that.